package.json

```diff
@@ -57,47 +55,26 @@
             "explorer/context": [
                 {
                     "command": "jupyter.openAsPairedNotebook",
-                    "when": "resourceLangId == python"
-                },
-                {
-                    "command": "jupyter.openAsPairedNotebook",
-                    "when": "resourceLangId == r"
-                },
-                {
-                    "command": "jupyter.openAsPairedNotebook",
-                    "when": "resourceLangId == julia"
-                },
-                {
-                    "command": "jupyter.openAsPairedNotebook",
-                    "when": "resourceLangId == fsharp"
-                },
-                {
-                    "command": "jupyter.openAsPairedNotebook",
-                    "when": "resourceLangId == csharp"
-                },
-                {
-                    "command": "jupyter.openAsPairedNotebook",
-                    "when": "resourceLangId == java"
-                },
-                {
-                    "command": "jupyter.openAsPairedNotebook",
-                    "when": "resourceLangId == powershell"
-                },
-                {
-                    "command": "jupyter.openAsPairedNotebook",
-                    "when": "resourceLangId == typescript"
-                },
+                    "when": "resourceLangId in vscode-jupytext.resourceLangIds"
+                }
+            ],
+            "editor/context": [
                 {
                     "command": "jupyter.openAsPairedNotebook",
-                    "when": "resourceLangId == javascript"
-                },
+                    "when": "resourceLangId in vscode-jupytext.resourceLangIds"
+                }
+            ],
+            "editor/title": [
                 {
                     "command": "jupyter.openAsPairedNotebook",
-                    "when": "resourceLangId == markdown"
-                },
+                    "when": "resourceLangId in vscode-jupytext.resourceLangIds",
+                    "group": "navigation"
```

Comment:
`"group": "navigation"` makes the command show up next to the icons to the right of the file tabs.  Since I didn't add an icon, it just shows the full "Open as Jupyter Notebook" command text